app/ports/models/buildhistory.py

```diff
@@ -0,0 +1,127 @@
+import urllib.request
+import ssl
+import json
+import datetime
+
+from django.db import models
+
+from MacPorts.config import BUILDERS_JSON_URL, BUILDBOT_URL_PREFIX, BUILDS_FETCHED_COUNT
+
+
+class Builder(models.Model):
+    name = models.CharField(max_length=100, db_index=True)
+
+
+class BuildHistory(models.Model):
+    builder_name = models.ForeignKey(Builder, on_delete=models.CASCADE)
```

Comment:
If we are storing the `id`, we should call this field `builder`, rather than `builder_name` (and then end up with `builder_name_id`) as the latter is confusing.

Reply:
I'm also confused about why we end up with two indices:
```
CREATE INDEX ports_build_builder_289e95_idx
    ON public.ports_buildhistory USING btree
    (builder_name_id)
    TABLESPACE pg_default;

CREATE INDEX ports_buildhistory_builder_name_id_9ac9bd18
    ON public.ports_buildhistory USING btree
    (builder_name_id)
    TABLESPACE pg_default;
```